docs/Docker/dockerLocal.md

```diff
@@ -52,9 +51,11 @@ git clone --recursive https://github.com/open-amt-cloud-toolkit/open-amt-cloud-t
 
 ## Build and Run the Docker Images
 
-The environment file (`.env`) now contains the MPS and RPS environment variables to pass to the Docker engine.
+Build the MPS, RPS, and Sample UI Docker images and launch the stack.
 
-1.  Build the MPS, RPS, and Sample UI Docker images and launch the stack
+**To build:**
+
+1.  Use docker-compose:
 
     ``` bash    
     docker-compose -f "docker-compose.yml" up -d --build
```

Comment:
@rsdmike Do we assume people have docker setup on linux to run rootless or do we default to including sudo as part of cmds?

Reply:
Let's table this. I'll check it in, and then we can come back to this.